deployment/ansible/roles/model-my-watershed.tiler/templates/systemd-mount.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Mount Node Modules to Prevent Race Conditions
+
+[Mount]
+What=/var/cache/node_modules
+Where=/opt/tiler/node_modules
+Type=none
+Options=bind
+
+[Install]
+{% if ['development', 'test'] | some_are_in(group_names) -%}
```

Comment:
This conditional can be eliminated because this unit should only exist in Vagrant based environments. We just need this portion:

```
WantedBy=opt-tiler.mount
```